module.info

```diff
@@ -1,6 +1,6 @@
 Name: JIRA
-Version: 1.2.2
-Depends: monitoring (>=2.9.0), icingadb, ipl (>=0.11.0)
+Version: 1.3.0
+Depends: monitoring (>=2.9.0), icingadb (>= 1.0.0), ipl (>=0.11.0)
```

Comment:
please use the new `Requires` syntax like [here](https://github.com/Icinga/icingaweb2-module-cube/blob/master/module.info)